plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -34,7 +34,7 @@ func calculateScore(requested int64, capacity int64, node string) int {
 		return 0
 	}
 	if requested > capacity {
-		glog.Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
+		glog.V(4).Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
```

Comment:
Sounds like a good idea. I'll check to see if there is an issue already created for that.